build.sbt

```
@@ -16,11 +16,11 @@
 val shared = Seq(
   // Info
   organization := "com.github.asana.socko-asana-fork",
-  version      := "0.6.2",
-  crossScalaVersions := Seq("2.11.12", "2.12.7"),
+  version      := "0.6.3",
+  crossScalaVersions := Seq("2.12.7"),
```

should we bump this to `2.12.10` for posterity's sake?
```suggestion
  crossScalaVersions := Seq("2.12.10"),
```